packages/tasit-sdk/lib/index.js

```diff
@@ -0,0 +1,28 @@
+"use strict";
+
+Object.defineProperty(exports, "__esModule", {
+  value: true
+});
+exports.default = exports.Account = void 0;
+
+var _wallet = require("ethers/wallet");
+
+const create = async () => {
```

Comment:
@marcelomorgado Do you think it matters that we have this as an async function when `Wallet.createRandom` from ethers isn't?